frontend/layouts/OnboardingLayout/OnboardingLayout.jsx

```diff
@@ -59,13 +58,7 @@ const OnboardingLayout = ({ children, currentStep }) => {
   if (isTabletScreen) return <AppDisabled head={renderHead()} />;
 
   return (
-    <Grid
-      {...styles.mainContainer}
-      sx={{
-        background: (theme) =>
-          theme.palette.Background[themeState.data.background],
-      }}
-    >
+    <Grid {...styles.mainContainer}>
```

Comment:
Removed these variables as the theme is no longer being updated based on redux state